ckanext/switzerland/templates/emails/subscribe_manage.html

```diff
@@ -0,0 +1,151 @@
+<body style="line-height: 200%; font-family: sans-serif; padding-top: 0">
+<p>Guten Tag</p>
+
+<p>{{ site_title }} subscription options<br/>
+
+<p>
+    To manage subscriptions for {{ email }}, click this link:<br/>
+    <a href="{{ manage_link }}">{{ manage_link }}</a>
+</p>
+
+<p>Freundliche Grüsse<br/>Team Geschäftsstelle OGD</p>
+
+<p>
+    Geschäftsstelle Open Government Data<br/>
+    Bundesamt für Statistik BFS<br/>
+    Espace de l'Europe 10<br/>
+    CH-2010 Neuchâtel<br/>
+    <a href="www.bfs.admin.ch/ogd">www.bfs.admin.ch/ogd</a>
+</p>
+
+<p>
+    <a href="https://opendata.swiss">
+        <img src="https://opendata.swiss/images/logo_horizontal.png" alt="opendata.swiss"
+             width="420" style="max-width: 100%; height: auto;"/>
+    </a>
+</p>
+<p>
+    <a href="https://twitter.com/opendataswiss">
+        <img src="https://opendata.swiss/images/twitter.svg" alt="Twitter"
+             style="color: #fff; background-color: #009688; border: 0;"/>
+    </a>
+</p>
+
+<p>
+    <a href="{{ manage_link }}">Mein Abonnement verwalten</a>
+</p>
+
+--
+
+<p>Bonjour</p>
+
+<p>{{ site_title }} subscription options<br/>
+
+<p>
+    To manage subscriptions for {{ email }}, click this link:<br/>
+    <a href="{{ manage_link }}">{{ manage_link }}</a>
+</p>
+
+<p>Cordialement,<br/>Votre Team Secrétariat OGD</p>
+
+<p>
+    Secrétariat Open Government Data<br/>
+    Office fédéral de la statistique<br/>
+    Espace de l'Europe 10<br/>
+    CH-2010 Neuchâtel<br/>
+    <a href="www.bfs.admin.ch/ogd">www.bfs.admin.ch/ogd</a>
+</p>
+
+<p>
+    <a href="https://opendata.swiss">
+        <img src="https://opendata.swiss/images/logo_horizontal.png" alt="opendata.swiss"
+             width="420" style="max-width: 100%; height: auto;"/>
+    </a>
+</p>
+<p>
+    <a href="https://twitter.com/opendataswiss">
+        <img src="https://opendata.swiss/images/twitter.svg" alt="Twitter"
+             style="color: #fff; background-color: #009688; border: 0;"/>
+    </a>
+</p>
+
+<p>
+    <a href="{{ manage_link }}">Gérer mes abonnements</a>
+</p>
+
+--
+
+<p>Hello</p>
+
+<p>{{ site_title }} subscription options<br/>
+
+<p>
+    To manage subscriptions for {{ email }}, click this link:<br/>
+    <a href="{{ manage_link }}">{{ manage_link }}</a>
+</p>
+
+<p>Best regards,<br/>Team Open Government Data Office</p>
+
+<p>
+    Open Government Data Office<br/>
+    Federal Statistical Office<br/>
+    Espace de l'Europe 10<br/>
+    CH-2010 Neuchâtel<br/>
+    <a href="www.bfs.admin.ch/ogd">www.bfs.admin.ch/ogd</a>
+</p>
+
+<p>
+    <a href="https://opendata.swiss">
+        <img src="https://opendata.swiss/images/logo_horizontal.png" alt="opendata.swiss"
+             width="420" style="max-width: 100%; height: auto;"/>
+    </a>
+</p>
+<p>
+    <a href="https://twitter.com/opendataswiss">
+        <img src="https://opendata.swiss/images/twitter.svg" alt="Twitter"
+             style="color: #fff; background-color: #009688; border: 0;"/>
+    </a>
+</p>
+
+<p>
```

Comment:
Probably it was communicated this way,  but it is strange that in one email we are given 2 times link to manage subscriptions l.84 and. l.111

Reply:
The idea was that the email footer always contains the link to manage your emails - but actually, this email doesn't matter much right now, because it's the one that we won't send at the moment. You can see that it doesn't have proper translations either.

Tbh, I added the method and template for this email in our plugin mostly because I had so much trouble with unicode encoding/decoding, I wanted everything to be using the same methods so I wouldn't keep getting tripped up by encoding errors. :sweat:

Reply:
I could imagine, ok let us leave it like it is.